src/main/java/lotto/model/factory/LottoAutoFactory.java

```diff
@@ -0,0 +1,33 @@
+package lotto.model.factory;
+
+import static lotto.constant.LottoSetting.LOTTO_NUMBER_RANGE_MAX;
+import static lotto.constant.LottoSetting.LOTTO_NUMBER_RANGE_MIN;
+import static lotto.constant.LottoSetting.LOTTO_SIZE;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import lotto.model.number.LottoNumber;
+
+public class LottoAutoFactory {
+
+    private static final List<LottoNumber> rangeLottoNumbers = new ArrayList<>();
+
+    static {
+        IntStream.range(LOTTO_NUMBER_RANGE_MIN, LOTTO_NUMBER_RANGE_MAX + 1)
+            .forEach(i -> rangeLottoNumbers.add(new LottoNumber(i)));
```

Comment:
```suggestion
        IntStream.rangeClosed(LOTTO_NUMBER_RANGE_MIN, LOTTO_NUMBER_RANGE_MAX)
            .forEach(i -> rangeLottoNumbers.add(new LottoNumber(i)));
```
rangeClosed 를 활용해볼 수도 있겠네요 :)

Reply:
생각지도 못한 꿀팁 감사합니다 👍